spec/controllers/reports_controller_spec.rb

```diff
@@ -87,6 +87,61 @@
       end
 
     end
+
+    context "when reporting a comment (logged in)" do
+      before do
+        session[:user_id] = user.id
+      end
+
+      let(:comment) do
+        FactoryGirl.create(:comment, :info_request => info_request)
+      end
+
+      it "should mark the comment as having been reported" do
+        expect(comment.attention_requested).to eq(false)
```

Comment:
This is redundant – you can explicitly pass `:attention_requested => false` to the factory if its worth illustrating the default state of the subject. Same applies for all cases of this.